docs/user/tutorial/index.rst

```diff
@@ -579,32 +551,24 @@ as well as a plot similar to the one below.
 
    Traffic Analytics plot
 
+You can also download this data in :abbr:`CSV (Comma-Separated Values)`  format for closer inspection. 
+To do that, scroll to the bottom of the page
+and click the :guilabel:`Download all data` button.
+
 .. note::
 
-   The Traffic Analytics view explained above gives you a simple overview
-   of how your readers browse your documentation. It has the advantage that
-   it stores no identifying information about your visitors,
-   and therefore it respects their privacy.
-   However, you might want to get more detailed data by
+   You can get more detailed traffic data by 
    :ref:`enabling Google Analytics <analytics:Enabling Google Analytics on your Project>`.
-   Notice though that we take some extra measures to :ref:`respect user
+   Notice though that Read the Docs takes extra measures to :ref:`respect user
```

Comment:
Note: I noted this section in https://github.com/readthedocs/readthedocs.org/issues/9530, where we were discussing removing GA support.

Nothing to do here now, this is still accurate at the moment.